### ISSUE TITLE
this.supported is undefined when changing captions

### ISSUE THREAD

**Issue by ThibaultNocchi** [sampotts/plyr#2352]
### Expected behaviour
There shouldn't be an error in the console when changing tracks

### Actual behaviour
This console error is raised:
```
Uncaught TypeError: this.supported is undefined
    setup https://unpkg.com/plyr@3:1
    set currentTrack https://unpkg.com/plyr@3:1
    createMenuItem https://unpkg.com/plyr@3:1
    lt https://unpkg.com/plyr@3:1
    lt https://unpkg.com/plyr@3:1
    ge https://unpkg.com/plyr@3:1
    ge https://unpkg.com/plyr@3:1
    fe https://unpkg.com/plyr@3:1
    lt https://unpkg.com/plyr@3:1
    createMenuItem https://unpkg.com/plyr@3:1
    setCaptionsMenu https://unpkg.com/plyr@3:1
    update https://unpkg.com/plyr@3:1
    setTimeout handler*setup https://unpkg.com/plyr@3:1
    build https://unpkg.com/plyr@3:1
    wt https://unpkg.com/plyr@3:1
    <anonymous> https://cdpn.io/cp/internal/boomboom/pen.js?key=pen.js-ef204373-eaa0-7eac-0ca1-d47f23a19710:3
plyr@3:1:36562
```

### Steps to reproduce
* Open the console
* Go to the vanilla HTML5 demo link from the README (https://codepen.io/pen?template=bKeqpr)
* Change a caption, either via UI or via JS 

### Environment

- Browser: FF 93.0 and Chromium 94.0.4606.81
- Operating System: Linux

### Console errors (if any)
Given above

### Link to where the bug is happening
The basic demo displays it

Thanks in advance, have a nice day!

**Comment by tractorcow** [comment]
I encountered this issue on 3.6.12, and attempted a slight downgrade. 3.6.8 did not have this issue, if that helps narrow down the search range.

**Comment by tractorcow** [comment]
Perhaps it could be related to https://github.com/sampotts/plyr/pull/2261

**Comment by tractorcow** [comment]
Fixed with https://github.com/sampotts/plyr/pull/2399

**Comment by sampotts** [comment]
Thanks for the teamwork on this. It's released in v3.7.0 🎉
